lib/rubygems/test_case.rb

```diff
@@ -357,7 +357,12 @@ def setup
     Dir.chdir @tempdir
 
     ENV['HOME'] = @userhome
+    FileUtils.mkdir_p File.join(@userhome, ".gem")
+    File.write File.join(@userhome, ".gemrc"), "--- {}"
```

Comment:
Following the same criteria as in my other comment. Why are we creating this file after every test? Can we remove this and instead restore the lines in `test/rubygems/test_gem_commands_build_command.rb`, `test/rubygems/test_gem_commands_cert_command.rb` and so on that were creating this file and/or folder only when needed?

Reply:
I understood your concerns. But if I removed them, over the 50+ tests was failed because it depends on the current `.gem` location or others. I choose the default fallback way on this time. 

I'm happy to fix with https://github.com/rubygems/rubygems/pull/3545#discussion_r417232784